packages/tools/examples/videoTools/index.ts

```diff
@@ -191,16 +211,27 @@ async function run() {
       {
         mouseButton: MouseBindings.Auxiliary, // Middle Click
       },
+      {
```

Comment:
Tweaked the bindings so that it would work without a mouse or with a mouse effectively.